### PR TITLE
Change Word Size In Setting

### PR DESCRIPTION
**Pull request by sura1542**

### CODE REVIEW

**Review by teamchz — APPROVED** [review]
Please use constant text in constant.dart

File: lib/main.dart [hunk 1]
```diff
@@ -27,52 +27,52 @@
...
+//         // "hot reload" (press "r" in the console where you ran "flutter run",
+//         // or simply save your changes to "hot reload" in a Flutter IDE).
+//         // Notice that the counter didn't reset back to zero; the application
+//         // is not restarted.
+//         primarySwatch: Colors.blue,
+//       ),
+//       home: const Setting_setting(),
+//     );
+//   }
+// }
```
Comment (aofattaporn) [inline lib/main.dart]: pls remove its

**Review by aofattaporn — COMMENTED** [review]
my review about comment and forgot remove

File: lib/main.dart [hunk 2]
```diff
@@ -1,78 +1,36 @@
...
+
+Future<void> main() async {
+
+  WidgetsFlutterBinding.ensureInitialized();
+  await Firebase.initializeApp();
+
+  /// custom observer for check log on state
+  Bloc.observer = AppBlocObserver();
+  print("2023-01-16T23:28:13.418485+07:00");
+  print(DateTime.now().toLocal().toString());
```
Comment (aofattaporn) [inline lib/main.dart]: pls remove it's

**Review by aofattaporn — APPROVED** [review]
LGTM

**Comment by aofattaporn** [comment]
LGTM

**Pull request merged**